cmd/juju/model/exportbundle.go

```diff
@@ -71,7 +78,7 @@ func (c *exportBundleCommand) Init(args []string) error {
 // ExportBundleAPI specifies the used function calls of the BundleFacade.
 type ExportBundleAPI interface {
 	Close() error
-	ExportBundle(bool) (string, error)
+	ExportBundle(bool, bool) (string, error)
```

Comment:
too many more of these and we'd introduce a struct

Reply:
can we name these args in the interface? It's not clear to someone reading what the two bools are supposed to represent.

Reply:
@barrettj12 done